cor-journey-test-data/src/main/resources/tps/testdata/multipayment/journey-1-With5itemsInBasket.json

```diff
@@ -1,7 +1,7 @@
 {
   "_id" : "64886ed616fe8b501cbf0088",
   "journeyState" : {
-    "Landing" : { }
+    "Started" : { }
```

Comment:
I do wonder whether Started is misleading for journey state when the journey is on the landing page and not just a started journey